README.md

```diff
@@ -1,18 +1,28 @@
 # Evals
 
-Evals is a framework for evaluating OpenAI models and an open-source registry of benchmarks.
+Evals is a framework for evaluating large language models (LLMs) and LLM Programs. It also includes an open-source registry of challenging benchmark evals.
 
 You can use Evals to create and run evaluations that:
 - use datasets to generate prompts,
-- measure the quality of completions provided by an OpenAI model, and
-- compare performance across different datasets and models.
-
-With Evals, we aim to make it as simple as possible to build an eval while writing as little code as possible. To get started, we recommend that you follow these steps **in order**:
-1. Read through this doc and follow the [setup instructions below](README.md#Setup).
-2. Learn how to run existing evals: [run-evals.md](docs/run-evals.md).
-3. Familiarize yourself with the existing eval templates: [eval-templates.md](docs/eval-templates.md).
-4. Walk through the process for building an eval: [build-eval.md](docs/build-eval.md)
-5. See an example of implementing custom eval logic: [custom-eval.md](docs/custom-eval.md).
+- measure the quality of completions provided by LLMs or LLM programs, and
+- compare performance across different datasets.
+
+(April 7 update): We now support evaluating the performance of any end-to-end application logic via the [Completion Function Protocol](docs/completion-fns.md).
```

Comment:
`performance of any LLM program` instead? (assuming we go with LLM programs)

Reply:
How about
"We now support evaluating the performance of any prompt completion logic, like LLMs, prompt chains or tool-using agents, via the [Completion Function Protocol](docs/completion-fns.md)."